packages/config/tailwind-preset.js

```diff
@@ -58,8 +58,9 @@ module.exports = {
       borderColor: {
         emphasis: "var(--cal-border-emphasis, #9CA3AF)",
         default: "var(--cal-border, #D1D5DB)",
-        subtle: "var(--cal-border-subtle, #E5E7EB)",
+        subtle: `var(--cal-border-subtle, ${subtleColor})`,
         muted: "var(--cal-border-muted, #F3F4F6)",
+        booker: `var(--cal-border-booker, ${subtleColor})`,
```

Comment:
booker was using subtle so use the same colour here so that by default the border color remains same.